### DISCUSSION TITLE
Daily 6/10

### DISCUSSION THREAD

**Discussion by rpgrca** [rpgrca/proyecto-ispc-fullstack#96] [General]
Ayer estuve muy ocupado en el trabajo así que no pude avanzar mucho. Hoy tengo reunión presencial en el trabajo así que hasta la tarde no me libero. También en base a lo que leí en el foro se pide algo de la parte de back-end por lo que me voy a poner a pensar en diagramar la estructura de datos que podemos llegar a usar. Las voy a crear como issues y voy a colocar lo que diagrame en ellas.

**Comment by NataliaAlvarezIspc** [comment]
Ayer estuve trabajando con los formularios de registración y de login y los subi a github. Los dejo para que los revisen.
Hoy voy a participar de los encuentros sincrónicos y estaré estudiando SGC y leyendo el nuevo material de Full Stack.

**Comment by AhumadaCeleste** [comment]
Ayer estuve trabajando en issues, estudie un poco de SGC. Hoy veré las clases de SGC y fullStack de ayer, intentare completar mas issue y estudiar sgc. Mañana espero poder rendir SGC y retomar Cisco pra terminar modulo 3 de la segunda parte.
